keyboards/planck/keymaps/vifon/keymap.c

```diff
@@ -55,14 +57,14 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 [_PP] = { /* Pure Pro */
```

Comment:
All your layers should be formatted to use an appropriate LAYOUT macro, as seen here:

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/keyboards/planck/keymaps/default/keymap.c#L58-L63